### ISSUE TITLE
Test file output broken

### ISSUE THREAD

**Issue by robbiepitts** [glimmerjs/glimmer-build#23]
I updated glimmer deps in some glimmerjs org projects to their latest versions (from 0.21.x to 0.22.0) and this happened. I get the error in other projects that use @glimmer/build as well, not just @glimmer/di:

```
$ npm run test

> @glimmer/di@0.1.8 test /Users/rpitts/Repos/personal/glimmer-di
> testem ci

not ok 1 Firefox 48.0 - Global error: Error: Could not find module `node_modules/@glimmer/util/index` imported from `src/registry`
```

Indeed, src/registry is importing an invalid path:

```
define("src/registry", ["require", "exports", "node_modules/@glimmer/util/index"] // ...
```

**Comment by dgeb** [comment]
Hmmm ... the main glimmer repo has undergone quite a lot of churn recently. We'll need to investigate. I have doubts whether this dependency is necessary even (just for the `Dict` interface I think).

**Comment by robbiepitts** [comment]
@dgeb Sorry, just updated the issue with a little more detail. This happens in @glimmer/component as well. I haven't tried it with the others but I bet it's the same deal.

**Comment by robbiepitts** [comment]
Here is the module def for registry without updating @glimmer/util in @glimmer/di:

```
define("src/registry", ["require", "exports", "@glimmer/util"] // ...
```

**Comment by dgeb** [comment]
So I diff'd the distributions of @glimmer/util 0.21 and 0.22. Although both versions have a `typings` setting that points to `index.ts`, only 0.22 actually contains this file in the root dir. I still don't grok why TSC is mangling this path when doing AMD builds :/

**Comment by dgeb** [comment]
Fixed via #24
